frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -327,6 +380,14 @@ export const dashboardLogic = kea<dashboardLogicType>([
         ],
     })),
     reducers(({ props }) => ({
+        _dashboardLoading: [
```

Comment:
the kea loaders `dashboardLoading` wasn't being set to true `loadDashboard` was triggered in `afterMount`. Not sure if someone's noticed a similar issue with the loaders plugin. Can't quite figure how my change would break it especially because this reducer works as expected.